### ISSUE TITLE
How could I add Chinese language

### ISSUE THREAD

**Issue by furick** [recalbox/recalbox-os#481]
The recalbox is cool and I really enjoy it. The only issue for me is that I am a Chinese and every rom I download from our website is Chinese. So the files' names in recalbox are all square. I want to join this project and how could I add Chinese language. I believe many Chinese players are waiting for it. :)

**Comment by MikaXII** [comment]
Hmm maybe here : 
https://github.com/digitalLumberjack/recalbox-buildroot/blob/unified/configs/recalbox-rpi2_defconfig#L14
and add : 
* zh_CN.UTF-8 <- Simplified 
* zh_TW.UTF-8 <- Traditionnal

We should add this on recalbox-rpi1 too.

There are already one user who work on chinese translation, see here : 
https://github.com/digitalLumberjack/recalbox-emulationstation/pull/36

**Comment by digitalLumberjack** [comment]
Hi, the PR digitalLumberjack/recalbox-emulationstation#36 lacks of some strings.

If you update it to the last revision of https://github.com/digitalLumberjack/recalbox-emulationstation/blob/recalbox-buildroot/locale/lang/fr/LC_MESSAGES/emulationstation2.po we could add chinese in a next release.

You also can check if the translation is ok with the pr

**Comment by MikaXII** [comment]
We shouldn't add the chinese locale for the next release ? Because every rom file stay like □□□□ when they was read by ES no ?

**Comment by digitalLumberjack** [comment]
Yes of  course @MikaXII  i just answered to @furick

**Comment by furick** [comment]
@MikaXII  @digitalLumberjack  really a good news for me! Thanks for all your efforts.

**Comment by MikaXII** [comment]
@furick [beta-15](https://github.com/recalbox/recalbox-os/releases/tag/v3.3.0-beta15) tell us if that work :)

**Comment by furick** [comment]
Yeah~ it works well. The files' name and the translation are both OK.
Really appreciate for your job!
I just find a little bug using Mac OS X to copy files into the SD card. If
I tried to copy the files which names are Chinese in this way, they were
still ??? in the recalbox game list. But I tried to copy files via Windows
Network,  the file names are OK.

2015-11-25 19:13 GMT+08:00 MikaXII <notifications@github.com>:

**Comment by furick** [comment]
Hi Mika
Yeah~ it works well. The files' name and the translation are both OK.
Really appreciate for your job!
I just find a little bug using Mac OS X to copy files into the SD card. If
I tried to copy the files which names are Chinese in this way, they were
still ??? in the recalbox game list. But I tried to copy files via Windows
Network,  the file names are OK.

2015-11-25 19:13 GMT+08:00 MikaXII <notifications@github.com>:
